optimum/intel/openvino/modeling_diffusion.py

```diff
@@ -1378,6 +1445,18 @@ class OVModelVae:
     def __init__(self, decoder: OVModelVaeDecoder, encoder: OVModelVaeEncoder):
         self.decoder = decoder
         self.encoder = encoder
+        self.spatial_compression_ratio, self.temporal_compression_ratio = None, None
+        if hasattr(self.decoder.config, "spatio_temporal_scaling"):
+            patch_size = self.decoder.config.patch_size
+            patch_size_t = self.decoder.config.patch_size_t
+            spatio_temporal_scaling = self.decoder.config.spatio_temporal_scaling
+            self.spatial_compression_ratio = patch_size * 2 ** sum(spatio_temporal_scaling)
+            self.temporal_compression_ratio = patch_size_t * 2 ** sum(spatio_temporal_scaling)
+        self.latents_mean, self.latents_std = None, None
+        if hasattr(self.decoder.config, "latents_mean_data"):
+            self.latents_mean = torch.tensor(self.decoder.config.latents_mean_data)
+        if hasattr(self.decoder.config, "latents_std_data"):
+            self.latents_std = torch.tensor(self.decoder.config.latents_std_data)
```

Comment:
This should rather be done in the encoder/decoder using register_to_config (see https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/autoencoders/autoencoder_kl_ltx.py#L1144), the wrapper class is just interfacing the two components.

Reply:
not sure that I understand you. In VAE, `latents_std` and `latents_mean` are trainable parameters, we can not load them using the same way like pytorch because in general case, we have not access to pyotrch model weights.  To handle that I register data of these tensors in config and adds to vae manually

Reply:
@IlyasMoutawwakil  any ideas or comments?

Reply:
makes sense, thanks for the explanation !